time now
std_msgs/Float64MultiArray distance_vector
std_msgs/Float64MultiArray stiffness_vector
baxter_core_msgs/JointCommand torque
geometry_msgs/Vector3[] force
#std_msgs/Float32 error
